docs/developer-docs/latest/development/data-management.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Data Management System - Strapi Developer Docs
+description: Import, export, and transfer data using the Strapi CLI
+sidebarDepth: 3
+canonicalUrl: https://docs.strapi.io/developer-docs/latest/development/export-import.html
+---
+
+# Data Management System <BetaBadge />
+
+:::callout 🚧 Feature under development
+The data management system is under development. Not all use cases are covered by the initial release. You can provide feedback about desired functionality on the [Strapi feedback website](feedback.strapi.io).
+:::
+
+Occasionally you need to move data out of or into a Strapi instance. The data management system allows you to efficiently extract data from an existing instance or archive and import that data into a separate instance. Strapi provides CLI-based commands that allow you to export and import data. Common use cases include:
+
+- creating a data backup,
+- restoring data from a backup.
+
+ The following documentation details examples of how to use the `strapi export` and `strapi import` commands.
+
+:::strapi Using the Command Line Interface (CLI)
+The `strapi export` and `strapi import` CLI commands with all of the available options are listed in the [Command Line Interface documentation](/developer-docs/latest/developer-resources/cli/CLI#strapi-export.md).
+:::
+
+## Export data using the CLI tool
+
+The `strapi export` command by default exports data as an encrypted and compressed `tar.gz` file. The default export command exports:
+
+- the project configuration,
+- entities: all of your content,
+- links: relations between your entities
+- assets: files stored in the uploads folder,
+- schemas,
+- `metadata.json` file.
```

Comment:
```suggestion
- the `metadata.json` file.
```
WDYT? For consistency with `the project structure` at the beginning of the list.